recipes/phyfleaux/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 R. Tohid
+#
+# Distributed under the Boost Software License, Version 1.0. (See a copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+FROM ubuntu:18.04
+
+RUN apt update
+RUN apt install -y sudo
+
+# add user
+RUN adduser --disabled-password --gecos '' stellar
+RUN adduser stellar sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# OS Packages
+RUN apt install -y python3.8 python3.8-dev python3.8-distutils
+RUN apt install -y build-essential
+RUN apt install -y cmake automake autoconf libtool lld
+RUN apt install -y libpng-dev libjpeg-dev
+RUN apt install -y zlib1g-dev libz-dev
+RUN apt install -y git vim
+RUN apt install -y wget curl
+RUN apt install -y bash-completion
+RUN apt install -y llvm-8 llvm-8-dev clang-8
+
+# install pip3.8
+WORKDIR /tmp/
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+RUN CC=clang-8 CXX=clang++-8 python3.8 get-pip.py
```

Comment:
Why aren't ENV CC and ENV CXX set for the while file instead of being replicated many times?

Reply:
Will do.

Reply:
[Updated](https://github.com/STEllAR-GROUP/docker-containers/blob/62087813ade29ca57b2eab0bfec14cc7a10c60ad/recipes/phyfleaux/Dockerfile.pytiramisu0)